### PR TITLE
Supplier additional contact detail

### PR DESCRIPTION
**Pull request by kiproping**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #55 (partially)

## The problem

Supplier additional contact detail model was missing

## How I resolved it

Added the missing model

## How to verify it

testing

Notify the following people: @darrynten

### CODE REVIEW

File: tests/SageOne/Models/SupplierAdditionalContactDetailTest.php [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        $this->assertEquals(1, $allAdditionalContactDetails->results[0]->id);
+        $this->assertEquals(7, $allAdditionalContactDetails->results[0]->supplierId);
+        $this->assertEquals('sample string 3', $allAdditionalContactDetails->results[0]->contactName);
+        $this->assertEquals('sample string 4', $allAdditionalContactDetails->results[0]->designation);
+        $this->assertEquals('sample string 5', $allAdditionalContactDetails->results[0]->telephone);
+        $this->assertEquals('sample string 6', $allAdditionalContactDetails->results[0]->fax);
+        $this->assertEquals('sample string 7', $allAdditionalContactDetails->results[0]->mobile);
+        $this->assertEquals('sample@string.com', $allAdditionalContactDetails->results[0]->email);
+
+        $this->assertEquals(2, $allAdditionalContactDetails->results[1]->id);
```
Comment (fergusdixon) [inline tests/SageOne/Models/SupplierAdditionalContactDetailTest.php:184]: These tests don't make sense as it would only get supplier details for ID: 7.

**Review by fergusdixon — DISMISSED** [review]
One of the tests need to be changed as a request for ID: 7 shouldn't return another model with a different ID

File: tests/SageOne/Models/SupplierAdditionalContactDetailTest.php (hunk 1 above)
Comment (fergusdixon) [reply tests/SageOne/Models/SupplierAdditionalContactDetailTest.php:184]: The mocks need to be changed to reflect that too

Comment (kiproping) [reply tests/SageOne/Models/SupplierAdditionalContactDetailTest.php:184]: That's the point. It's supposed to `Get the Additional Contacts listing for a specific Supplier`. according to the documentation here https://accounting.sageone.co.za/api/1.1.2/Help/Api/GET-SupplierAdditionalContactDetail-GetAdditionalContactDetails-id. So I am limiting the results to a specific supplier.

Comment (fergusdixon) [reply tests/SageOne/Models/SupplierAdditionalContactDetailTest.php:184]: Sorry, misread the docs

**Review by darrynten — APPROVED** [review]
Looks good to me

**Pull request merged**